src/_utils/_git_secret_tools_linux.sh

```diff
@@ -8,8 +8,7 @@ function __replace_in_file_linux {
 
 
 function __temp_file_linux {
-  local filename
-  filename=$(mktemp)
+  local filename=$(mktemp)
```

Comment:
I guess this will break `shellcheck`. That was a weird rule.
I can not fully explain why it is required.